data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,328 @@ mission "Coalition: Outskirts Gauger Ready"
+			`	"Wait, really?" asks Anna.`
+			`	You continue to explain the Coalition to Anna for a while, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them.`
+			choice
+				`	"By any chance, are you afraid of centaurs, beetles, and spiders?"`
+			`	"No, not at all. I'm not afraid of them. Why do you ask?"`
+			choice
+				`	"Because the Coalition are a race of three alien species, who look like spiders, beetles, and centaurs.`
+			`	"Interesting. Their stories might just inspire my next novel. But, they wouldn't speak English. How are they going to read my books?"`
+			choice
+				`	"They have a translation device."`
+				`	"I'm sure they can learn the language."`
+			`	"That's reassuring... but, how am I even going to get there?"`
+			choice
+				`	"I'll take you there."`
+			`	"Do I have to pay anything? As a struggling author, I really can't pay you."`
+			choice
+				`	"It's okay. I'm willing to make an exception for you."`
+			`	"Thank you so much! Just wait for a while, I need to finish my book signing event. It should be roughly 45 minutes before it's over, and then we can go to whatever this Coalition thing is," says Anna.`
+			`	You reply. "Good luck. By the way, we're going to <destination> in Coalition space.`
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk. You prepare to leave the spaceport.`
+				accept
+	on complete
+		conversation
+			`	You leave the ship with Anna following you. The moment you are out of your ship, three Kimek surround you and Anna. The Kimek in the middle speaks, and then brief silence. The silence is broken as the yellow box speaks.`
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"` # heliarch
+			`	Anna clears her throat. "I don't know who you are, but my name is Anna Fangvin."`
+			`	Once Anna finishes her introduction, you speak. "She is my friend. Back in my space, she hasn't been a successful author. I brought her here, hoping she would find success writing books for the Coalition.", you explain.`
+			`	"What kind of books does she write?", asks the Kimek.` # heliarch
+			`	"Oh, I write science fiction books," says Anna.`
+			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
+			`	"Visit Coalition space, we will allow Anna to. Alongside you, <first>, welcome here, she is, for as long as no trouble, you cause. But first, register she must."`
+			`	"Register?", asks Anna.`
+			`	"You have to register yourself with the Coalition before you can do most stuff here. I'll take you to the Ring of Friendship, that's where you can complete the registration process."`
```

Comment:
Same thing as before, just have the player explanation narrated, without quoted dialog from the player character.